sprintly/sprintly-kanban
# 9285 style cards

#### What's this PR do?

Adds Sprintly-ish styling to 5columns.
#### Where should the reviewer start?

item-card.js, item-card.less, columns.less
#### How should this be manually tested?

Load up and take a look -  make sure to resize browser and click clickable items (item score, tags, ..., etc.)
#### Additional info you want to provide?

There are some stubbed functions in here that I can either rip out or I can use this as an interstitial branch....which ever works best
#### What ticket or issue # does this fix?

Fixes #9285
#### Screenshots (if appropriate)

![screen shot 2015-03-16 at 3 56 31 pm](https://cloud.githubusercontent.com/assets/1137259/6678217/233ad1d8-cbf5-11e4-8a04-0fda70fe25c6.png)

![screen shot 2015-03-16 at 3 56 42 pm](https://cloud.githubusercontent.com/assets/1137259/6678218/2f64015a-cbf5-11e4-8054-8b2dd1e6a02a.png)

![screen shot 2015-03-16 at 3 56 55 pm](https://cloud.githubusercontent.com/assets/1137259/6678219/333cacf0-cbf5-11e4-8ee0-d9cd90f4d9ad.png)

![screen shot 2015-03-16 at 3 57 02 pm](https://cloud.githubusercontent.com/assets/1137259/6678220/369d1024-cbf5-11e4-9f62-35629594ee47.png)
#### What gif best describes this PR or how it makes you feel?

![200 copy 2](https://cloud.githubusercontent.com/assets/1137259/6678227/5fd86e3e-cbf5-11e4-9c97-a929b654cf4a.gif)
#### Definition of Done:
- [ ] Does this contain breaking changes?
- [ ] Did you bump the version number accordingly?
- [ ] Will this feature require a new piece of infrastructure be implemented?
- [ ] Did you update the documentation per these changes?
